src/experimental/riff-raff-yaml-file/deployments/cloudformation.ts

```diff
@@ -49,24 +49,33 @@ export function cloudFormationDeployment(
 
 export function addAmiParametersToCloudFormationDeployment(
   cfnDeployment: RiffRaffDeployment,
-  asg: GuAutoScalingGroup
+  autoScalingGroups: GuAutoScalingGroup[]
 ): RiffRaffDeploymentProps {
-  const { imageRecipe, app, amiParameter } = asg;
+  const amiParametersToTags = autoScalingGroups.reduce((acc, asg) => {
+    const { imageRecipe, app, amiParameter, imageRecipeEncrypted = true } = asg;
```

Comment:
Riff-Raff's [default](https://github.com/guardian/riff-raff/blob/73c7eac159fc36712e07783028a1cd7569010a78/magenta-lib/src/main/scala/magenta/deployment_type/CloudFormationDeploymentTypeParameters.scala#L84-L102) is to not use encrypted AMIs, so this default is a change from that. I think it's the direction we want to head in though, and we should move to make Riff-Raff default to `true` too.

Reply:
Update: These changes have been [reverted](https://github.com/guardian/cdk/pull/1697/commits/1333a22e3bcc96c3abc68d61c8c7196feb37ba20) as it appears [Riff-Raff doesn't follow](https://github.com/guardian/riff-raff/blob/3f580f70795c7b210e4085cb8d16fa95cfcd1321/magenta-lib/src/main/scala/magenta/deployment_type/CloudFormationDeploymentTypeParameters.scala#L161-L164) the `Encrypted` tag when set in `amiParametersToTags`. Recommend we add this change in a separate PR.